go.mod

```diff
@@ -20,6 +21,7 @@ replace (
 	k8s.io/controller-manager => k8s.io/controller-manager v0.27.4
 	k8s.io/cri-api => k8s.io/cri-api v0.27.4
 	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.27.4
+	k8s.io/dynamic-resource-allocation => k8s.io/dynamic-resource-allocation v0.28.2
```

Comment:
Just curious: is this to help with Goland's failures to run `go list`? I am experiencing a similar issue, and adding this line fixes it.

Reply:
Yes, that is exactly why it's in here.  I figured I'd clean it up with the rest of the deps once the rancher side merges.  Not sure if it's the "right" fix or not long-term.

Reply:
What's the issue with Goland and the need for this line?